natgeo/mortar
# SYTO-771: Create the POD Promo Component

https://jira.natgeo.com/browse/SYTO-771

This story adds in the base POD component. Any styling will be handled in a subsequent story ([SYTO-753](https://jira.natgeo.com/browse/SYTO-753)), but there is some base styling in place simply to help verify component functionality

To test:

- Pull SYTO-771-mortar3
- Verify unit tests pass by running `npm test`
- Run `gulp serve`
- Verify component at http://localhost:3000/components/podpromo/index.html using the configurator
- Verify component also renders as expected at http://localhost:3000/components/
